jbake-core/src/main/java/org/jbake/parser/MarkupEngine.java

```diff
@@ -32,6 +33,9 @@
 public abstract class MarkupEngine implements ParserEngine {
     private static final Logger LOGGER = LoggerFactory.getLogger(MarkupEngine.class);
 
+    public static final int MAX_HEADER_LINES = 50;
```

Comment:
This is a PoC so this would also be a config.
However, I think I will put the MarkupEngine changes aside from this PR.

Reply:
jap. good idea.

Reply:
try to limit changes that are needed for the TexyService thing. Please have a look at the failing tests and fix them.

Reply:
That's in my plan. Sorry for the noise, I was getting familiar with JBake. Now this is a bit mess but I will split soon, perhaps during the weekend, and update (reduce) this PR.